src/types/Clustering.ts

```diff
@@ -0,0 +1,8 @@
+export interface MeshCluster {
+  apiEndpoint: string;
+  isHomeCluster: boolean;
```

Comment:
What does this mean exactly?

Reply:
isHomeCluster is a flag to indicate that this is the cluster where Kiali/istio is installed.